docs/Azure_IoT_Integration_Guide/Azure_IoT_Hub_integration/Set_an_Observation.md

```diff
@@ -0,0 +1,123 @@
+# Set an Observation
+
+This section describes how to set an Observation in your Azure IoT Hub.
+
+Setting an Observation lets your devices know what value changes and for what particular resources they need to notify you about. An Observation can be set for any or all the components of the [data model](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Concepts/LwM2M_mappings/): objects, objects instances, and resources. Whenever there is a change in values, a device will send a Notify message to Coiote DM, which in turn will transfer it to Azure IoT Hub.
+
+In this section, you learn how to:
+
+  * Set an Observation on objects and resources
+  * Define Observation attributes
+  * See value changes for the observed objects and resources
+
+## Prerequisites
+
+1. An active Azure IoT Hub with hub owner access permissions.
+2. A Coiote DM user account with permissions to use the integration extension.
+3. A device group created in Coiote DM.
+4. [A configured extension between Coiote DM and Azure Iot Hub](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Azure_IoT_Hub_integration/Configuring_Azure_IoT_Hub_integration_extension/).
+5. [A configured integration template in Coiote DM](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Configure_integration_templates/Azure_integration_templates/).
```

Comment:
the template must be previously assigned to your device group, right? Lets mention it then

Reply:
good point, will add that